File: Evenorodd.cpp
#include<iostream>
using namespace std;
class Evenorodd{
    public
    :

    void check(int num)
    {
        if(num%2==0){
            cout<<"number is even";
        }
        else{
            cout<<"number is odd";
        }
    }
};
int main(){
    int num;
    Evenorodd check1;
    cout<<"enter a number:"<<endl;
    cin>>num;
    check1.check(num);
    return 0;
}

File: Evenorodd.java
import java.util.Scanner;
    public class Evenorodd{
        public void checknum(int number){
            if(number%2==0){
                System.out.println("Given number is even");
            }
            else{
                System.out.println("Given number is odd");
            }
        }
        public static void main(String args[]){
            System.out.println("Enter number to check even or odd:");
            Scanner svar=new Scanner(System.in);
            int number=svar.nextInt();
            Evenorodd check1=new Evenorodd();
            check1.checknum(number);
        }
}